modules/docker/host.go

```diff
@@ -0,0 +1,44 @@
+package docker
+
+import (
+	"os"
+	"strings"
+)
+
+// GetDockerHost returns the name or address of the host on which the Docker engine is running.
+func GetDockerHost() string {
+	return getDockerHostFromEnv(os.Environ())
+}
+
+func getDockerHostFromEnv(env []string) string {
+	// Parses the DOCKER_HOST environment variable to find the address
+	//
+	// For valid formats see:
+	// https://github.com/docker/cli/blob/6916b427a0b07e8581d121967633235ced6db9a1/opts/hosts.go#L69
+	var dockerUrl []string
+
+	for _, item := range env {
+		envVar := strings.Split(item, "=")
+		if envVar[0] == "DOCKER_HOST" && len(envVar) == 2 {
```

Comment:
NIT: I happen to know `strings.Split` returns a non-empty array, but since we're checking length anyway, it's safer to do that length check first.

```suggestion
		if len(envVar) == 2 && envVar[0] == "DOCKER_HOST" {
```